demos/index.html

```diff
@@ -8,14 +8,17 @@
 <dl>
   <dt><a href="append-child.html">Append child WritableStream</a>
   <dd>Piping through a series of transforms to a custom writable stream that appends children to
-  the DOM.
+  the DOM.</dd>
```

Comment:
The end tag is not needed, see https://html.spec.whatwg.org/multipage/grouping-content.html#the-dd-element.

I find it tidier to leave it out.